### DISCUSSION TITLE
Support import from G!Docs, including comments

### DISCUSSION THREAD

**Discussion by metasj** [knowledgefutures/pubpub#1375] [Ideas]
PubPub's quite useful now for personal shared documents, which I used to develop on Google Docs with a couple of colleagues.  I'd like to import those, ideally with comments and original timestamps.  Options:

. _ Offer a GDoc import for world-readable docs, that just imports the text. 
. = Import comments without proper date + attribution; let the editors change those
. + Create new users for GDocs commenters and assign the original timestamps for comments 
. ‡  Let the editor merge the new users w/ existing ones who have accounts in the pub community

**Comment by gabestein** [comment]
Hi everyone — I'm going in and converting some of the longer-lived feature request/idea issues from Issues to Discussion threads. There's no exact heuristic about what belongs in the forum vs. what belongs as an issue, so **when in doubt, please post wherever you see fit — we won't mind!** But in general, we'd like to make Issues a space for things that we're committed to building/fixing, and Discussions a space for most everything else. So, if you're reporting a bug or requesting we make a straightforward change to an existing feature, go ahead and log an issue. Otherwise, Discussions is likely a better bet.

So, in this case, we're moving it to discussions and inviting anyone who's also interested in the idea to comment!
